etc/scripts/deploy_observability_crds.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash -e
 
-kubectl apply -f etc/kubernetes/manifests/overlays/local/observability/crds/observabilities.observability.redhat.com-v1.yml 
+kubectl apply -f etc/kubernetes/manifests/overlays/local/observability/crds/observabilities.observability.redhat.com-v1.yml
```

Comment:
is it correct that such resources are from the local overlay ?

Reply:
ah ok, maybe we can have an it overlay and we can just do an kubectl apply-k of the overlay to get the it set-up, what do  you think ?

Reply:
@lburgazzoli doing in this one